### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by backtwobaysix**
Some cool stuff

### CODE REVIEW

File: GWToolboxdll/Modules/ToolboxSettings.cpp [hunk 1]
```diff
@@ -207,6 +209,7 @@ void ToolboxSettings::DrawSettingInternal() {
         {"Discord",&use_discord},
         {"Distance",&use_distance},
         {"Effect Monitor",&use_effect_monitor},
+        {"Latency Widget", &use_latency_widget},
```
Comment (3vcloud) [inline GWToolboxdll/Modules/ToolboxSettings.cpp:212]: This list is alpha order, pop it further down

File: GWToolboxdll/Widgets/LatencyWidget.cpp [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        ImGui::PushFont(GuiUtils::GetFont(GuiUtils::FontSize::widget_large));
+        ImGui::SetCursorPos(cur);
+        ImGui::TextColored(GetColorForPing(GetPing()), "%u ms", GetPing());
+        ImGui::PopFont();
+
+        ImVec2 size = ImGui::GetWindowSize();
+        ImVec2 min = ImGui::GetWindowPos();
+        ImVec2 max(min.x + size.x, min.y + size.y);
+        if (ctrl_pressed && ImGui::IsMouseReleased(0) && ImGui::IsMouseHoveringRect(min, max)) {
+            char buffer[256];
```
Comment (3vcloud) [inline GWToolboxdll/Widgets/LatencyWidget.cpp:46]: Make this a chat command so people can hotkey it

Comment (3vcloud) [reply GWToolboxdll/Widgets/LatencyWidget.cpp:46]: Also this doesn’t need to be such a massive buffer size

File: GWToolboxdll/Widgets/LatencyWidget.h [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+public:
+    static LatencyWidget& Instance() {
+        static LatencyWidget instance;
+        return instance;
+    }
+    uint32_t ping_ms = 0;
+
+    const char* Name() const override { return "Latency"; }
+    const char* Icon() const override { return ICON_FA_BEER; }
```
Comment (3vcloud) [inline GWToolboxdll/Widgets/LatencyWidget.h:25]: Wtf?

Comment (backtwobaysix) [reply GWToolboxdll/Widgets/LatencyWidget.h:25]: It had a warning on it about it not being defined which was preventing build, idk

File: GWToolboxdll/Widgets/LatencyWidget.cpp (hunk 2 above)
Comment (backtwobaysix) [reply GWToolboxdll/Widgets/LatencyWidget.cpp]: Done

File: GWToolboxdll/Widgets/LatencyWidget.h (hunk 3 above)
Comment (backtwobaysix) [reply GWToolboxdll/Widgets/LatencyWidget.h]: Change icon too

**Pull request merged**